pkg/kubectl/cmd/kustomize/kustomize.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kustomize
+
+import (
+	"errors"
+	"github.com/spf13/cobra"
+	"os"
+
+	"k8s.io/cli-runtime/pkg/kustomize"
+	"k8s.io/kubernetes/pkg/kubectl/util/i18n"
+	"k8s.io/kubernetes/pkg/kubectl/util/templates"
+	"sigs.k8s.io/kustomize/pkg/constants"
+	"sigs.k8s.io/kustomize/pkg/fs"
+)
+
+type kustomizeOptions struct {
+	kustomizationDir string
+}
+
+var (
+	kustomizeLong = templates.LongDesc(i18n.T(`
+Generate a set of API resources from somedir with a kustomization.yaml
```

Comment:
```
Print a set of API resources generated from instructions in
a kustomization.yaml file.

The argument must be the path to the directory containing
the file (specifying the filename is optional), or a git repository
URL with a path suffix specifying same with respect to the
repository root.
```

Suggest saying _print_ instead of _generate_ since it's perhaps less mysterious.

Reply:
updated. Thank you